### ISSUE TITLE
Change format of number ranges in our csv downloads

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/iRecord#1599]
Verifiers continue to be frustrated by Excel's insistence on interpreting abundance ranges as dates, e.g. "6-20" gets changed to "June 2020" when verifiers download records and open into Excel.

I know there are potentially technical solutions to this, and that users can import csv files into Excel and specify the data type rather than just double-clicking to open the csv, but that all adds barriers to verifiers being able to deal with data efficiently.

I think the simplest short-term solution is to change the format of our number ranges so that "6-20" becomes "6 to 20" and likewise for other ranges.

I propose to make edits to the following termlists:
- [Butterfly App Abundance Codes](https://warehouse1.indicia.org.uk/index.php/termlist/edit/145) (the codes are fine but the ranges given as synonyms need to change) - used in iRecord app and iRecord Butterflies app
- [Number Ranges](https://warehouse1.indicia.org.uk/index.php/termlist/edit/32) - used in iRecord app and elsewhere

@johnvanbreda @kazlauskis do you see any problems with making this change?

@DavidHepper this will also affect the abundance ranges used on the dragonfly forms, will that be a problem?

**Comment by kazlauskis** [comment]
The apps will upload the termlist options as warehouse numbers, so if you are only rewording the existing range labels in the warehouse for the download report, then we don't need to update the apps.

**Comment by DavidHepper** [comment]
Yes, Excel is particularly stupid because it does this even when the column is specifically set as type Text not General.
The alternative I recommend and use when uploading to NBN Atlas is character En Dash ( U+2013 = Alt+0150 = &ndash; ) which looks similar but doesn't confuse Excel.

**Comment by kitenetter** [comment]
Thanks @DavidHepper for the en-dash suggestion, which I've now applied to the two termlists mentioned above. Closing.

**Comment by kitenetter** [comment]
@johnvanbreda in the csv files we generate to send to NBN Atlas, and the csv files generated for user downloads, the number ranges are still being outputted with a normal hyphen, even though the same records are displaying in iRecord with an en dash. Is there more that needs to be done to ensure the en dash version propagates through the system?

**Comment by DavidHepper** [comment]
Are the exports set to output UTF-8 ? This also preserves accented characters in Scottish Gaelic placenames, ♂ (= U+2642) in comments, etc. (I made the same mistake recently, sending to NBN Atlas.)
